helm/prometheus-meta-operator/templates/configmap.yaml

```diff
@@ -41,6 +41,15 @@ data:
         {{- if .Values.prometheus.version }}
         version: {{ .Values.prometheus.version }}
         {{- end }}
+      {{- if (.Values.prometheusAgent).shardScaleUpSeriesCount (.Values.prometheusAgent).shardScaleDownPercentage }}
```

Comment:
What is this syntax?
I see parenthesis and multiple conditions but no operator between the different conditions, and I can imagine what it does but I can't be sure.

Reply:
This is the helm syntax to say that .Values.prometheusAgent can be nil :)

Reply:
This actually relies on the order of operations so it executes the inner loop before it executes the other one

Reply:
So it's like `{{- if .Values.prometheusAgent AND .Values.prometheusAgent.shardScaleUpSeriesCount AND .Values.prometheusAgent AND .Values.prometheusAgent.shardScaleDownPercentage }}` ?
I didn't know you could put several conditions in one `if` with no operator between.

Reply:
Oh, I guess that answers my question :rofl: 
![image](https://github.com/giantswarm/prometheus-meta-operator/assets/12008875/fe079cdb-cb6c-42bf-8955-2558ff457900)